File: integrations/java/app/revanced/integrations/youtube/patches/AutoRepeatPatch.java
package app.revanced.integrations.youtube.patches;

import app.revanced.integrations.youtube.settings.Settings;

@SuppressWarnings("unused")
public class AutoRepeatPatch {
    //Used by app.revanced.patches.youtube.layout.autorepeat.patch.AutoRepeatPatch
    public static boolean shouldAutoRepeat() {
        return Settings.AUTO_REPEAT.get();
    }
}

File: patches/src/main/kotlin/app/revanced/patches/youtube/misc/autorepeat/AutoRepeatPatch.kt
package app.revanced.patches.youtube.misc.autorepeat

import app.revanced.patcher.patch.bytecodePatch
import app.revanced.patches.youtube.misc.loopvideo.loopVideoPatch

@Deprecated("Patch was renamed", ReplaceWith("looVideoPatch"))
val autoRepeatPatch = bytecodePatch {
    dependsOn(loopVideoPatch)
}
